spec/features/suggested_salary_searches_spec.rb

```diff
@@ -0,0 +1,287 @@
+describe "Suggested salary searches" do
+  let(:filter_page) { PageObjects::Page::ResultFilters::Funding.new }
+  let(:results_page) { PageObjects::Page::Results.new }
+  let(:courses_url) do
+    "http://localhost:3001/api/v3/recruitment_cycles/2020/courses"
+  end
+
+  def default_query_for_location_search(radius:)
+    { l: 1, loc: "Cat Town", lq: "Cat Town", lat: 51.4980188, lng: -0.1300436, rad: radius }
+  end
+
+  def stub_subjects
+    stub_request(:get, "http://localhost:3001/api/v3/subject_areas?include=subjects")
+  end
+
+  def course_fixture_for(results:)
+    file_name = case results
+                when 0
+                  "empty_courses.json"
+                when 2
+                  "two_courses.json"
+                when 4
+                  "four_courses.json"
+                when 10
+                  "ten_courses.json"
+                end
+
+    File.new("spec/fixtures/api_responses/#{file_name}")
+  end
+
+  def suggested_search_count_parameters
+    results_page_parameters.reject do |k, _v|
+      ["page[page]", "page[per_page]", "sort"].include?(k)
+    end
+  end
+
+  def stub_suggested_across_england_with_salary_filter(number_of_results:)
+    stub_request(:get, courses_url)
```

Comment:
Could this be extracted into a `stub_courses(number_of_results:, query:)` method? That would maybe make it easier to see the difference between each of these stubs and dry things up a bit.